### DISCUSSION TITLE
Deployment is queuing Hobby Plan

### DISCUSSION THREAD

**Discussion by 0xw01f** [vercel/community#3431] [Help] [answered]
### Summary

Hello
I am a bit embarrassed to ask this because I saw a lot of similar questions, but looking at the answers didn't help, so I have to ask.

 One of my projects deploys, since yesterday I think.
It's status stays "Queuing",  even if I wait for hours.

The Vercel server status seems to be okay to me.

So maybe it's due to my plan, because I have the free plan (Hobby).

It's also possible that I don't know how to deploy.

Note: When I commit changes to my old projects to make them deploy, it doesn't work either

### Steps to Reproduce

I am trying to deploy any new project

**Comment by amyegan** [comment, marked as answer]
Hey @0xw01f. Are you still seeing deployments stuck in "Queuing" status? If you share the project id or the unique url of a recent successful deployment, then I can look into this for you 🙏

**Reply by 0xw01f** [reply]
I don't see it queuing anymore !
Thank you for your answer
